spring-cloud-dataflow-module-deployer-spi/src/main/java/org/springframework/cloud/dataflow/module/ModuleStatus.java

```diff
@@ -79,7 +79,7 @@ public DeploymentState getState() {
 		}
 
 		if (states.size() == 0) {
-			return DeploymentState.undeployed;
+			return DeploymentState.unknown;
```

Comment:
Did we make a consensus on this yet? This would also mean the default state for the undeployed stream to be `partial` I think.